components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -449,22 +449,22 @@ - (CGRect)mdc_frameAlignedVertically:(CGRect)frame
                            alignment:(UIControlContentVerticalAlignment)alignment {
   switch (alignment) {
     case UIControlContentVerticalAlignmentBottom:
-      return CGRectMake(frame.origin.x, CGRectGetMaxY(bounds) - frame.size.height, frame.size.width,
-                        frame.size.height);
+      return CGRectMake(frame.origin.x, CGRectGetMaxY(bounds) - CGRectGetHeight(frame), CGRectGetWidth(frame),
+                        CGRectGetHeight(frame));
 
     case UIControlContentVerticalAlignmentCenter: {
-      CGFloat centeredY = MDCFloor((bounds.size.height - frame.size.height) / 2) + bounds.origin.y;
-      return CGRectMake(frame.origin.x, centeredY, frame.size.width, frame.size.height);
+      CGFloat centeredY = MDCFloor((bounds.size.height - CGRectGetHeight(frame)) / 2) + bounds.origin.y;
```

Comment:
This should use `CGRectGetHeight` for the `bounds`  and `CGRectGetMinY` for the origin; it also needs to be broken-up (too long).

```objectivec
CGFloat centeredY =
    MDCFloor((CGRectGetHeight(bounds) - CGRectGetHeight(frame)) / 2)
    + CGRectGetMinY(bounds);
```